Dockerfile

```diff
@@ -21,11 +21,14 @@ RUN wget --quiet https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1
 RUN tar xvfj /opt/phantomjs-2.1.1-linux-x86_64.tar.bz2 && \
     ln -s /opt/phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/bin/phantomjs
 
-# Copy the codebase
-COPY . /demo-test-suite
+RUN mkdir /demo-test-suite
```

Comment:
I was trying to avoid creating a *file* called /demo-test-suite whose
contests look like the lines of requirements.txt; have I misunderstood the
semantics of "COPY"?
On Jun 22, 2016 09:26, "Erik Wiffin" <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/sync-for-science/test-suite/pull/24#discussion_r68051672>
> :
>
> > @@ -21,11 +21,14 @@ RUN wget --quiet https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1
> >  RUN tar xvfj /opt/phantomjs-2.1.1-linux-x86_64.tar.bz2 && \
> >      ln -s /opt/phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/bin/phantomjs
> >
> > -# Copy the codebase
> > -COPY . /demo-test-suite
> > +RUN mkdir /demo-test-suite
>
> What's the point of running mkdir instead of just starting by copying?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sync-for-science/test-suite/pull/24/files/706745e8ad5f40d75264f4b79b77659b65d4eb2a#r68051672>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AATHAXyFuoRAUWfWnrZkmXp3nzrjMFHJks5qOTflgaJpZM4I7WTA>
> .
>